sms/v1/openapi.yaml

```diff
@@ -81,6 +100,146 @@ paths:
         - "Send SMS"
       summary: Sends SMS via POST HTTP method
       operationId: sendSmsUsingPOST
+      parameters:
+        - name: to
+          in: query
+          description: Destination phone number in international phone format
+          schema:
+            type: string
+        - name: from
+          in: query
+          description: |
+            This parameter gives the identification of the sending party, which can either be the phone number in international format or an alphanumeric identifier with up to 11 characters.
+
+            `Some destination networks impose restrictions on the sender ID format. Please check the coverage list
+            and contact us for more information.`
+          required: true
+          schema:
+            type: string
+        - name: message
+          in: query
+          description: |
+            The message will be sent with characters encoded either in GSM standard alphabet (GSM-7) or GSM Unicode alphabet (UCS-2).
+
+            - GSM standard alphabet [GSM-7](http://en.wikipedia.org/wiki/GSM_03.38). Maximum length is 160 characters per single message and 153 characters
+            per concatenated message.
+            - GSM Unicode alphabet [UCS-2](http://en.wikipedia.org/wiki/GSM_03.38). Maximum length is 70 characters per single message and 66 characters
+            per concatenated message.
+
+            tyntec automatically splits the sent message into concatenated messages if the message
+            exceeds the given limits. These concatenated messages are unified by the handset once again and
+            they are displayed as one message (if supported by the handset).
+
+            `tyntec will charge for each part of the concatenated message as an individual message.`
+
+          required: true
+          schema:
+            type: string
+        - name: encoding
+          in: query
+          description: Encoding selection between GSM7, UNICODE or the default AUTO selection
+          schema:
+            type: string
+            enum:
+              - AUTO
+              - GSM7
+              - UNICODE
+            default: AUTO
+        - name: gateway
+          in: query
+          description: Which gateway to use for the delivery of the message
```

Comment:
Question: What does a _gateway_ imply? I think it's worth to state it somewhere. Might be an external link to a blog / documentation post of us.

Reply:
Yes if i knew what exactly we mean with "gateway". If you have a good explanation please share it with me and i will add.